docs/roadmap.markdown

```diff
@@ -4,9 +4,122 @@ title: "Roadmap"
 permalink: /roadmap/
 ---
 
-The Developer Toolkit is comprised of a set of tools that are worked on in
-collaboration with the community.  The following unified roadmap provides an
-overview of the current priorities across the toolkit, and links to the
-specific tool roadmaps.
+---
+---
+# Note that this is a work-in-progress, and subject to change!
+---
+---
+
+## Context
+
+The Developer Toolkit is a set of tools that simplify New Relic configuration and management. Enabling developers to implement Observability as Code through industry standard frameworks such as Terraform and CloudFormation, or integrate into their existing workflow with the New Relic CLI. For those who require fine-grained control, the New Relic Client takes all of the guesswork out of directly using New Relic APIs.
+
+<br/>
+
+Problem | Reasons | Solution / Benefit
+------- | ------- | ------------------
+| Configuring New Relic is time consuming. | <ul><li>Users must use the UI, or create custom API scripts<li>Not all configuration can be done through APIs<li>APIs are different based on New Relic product<li>Existing libraries to reduce effort suffer from lack of ownership / maintenance</ul> | Provide tools to configure New Relic that reduces effort of the user |
+| New Relic configuration can be inconsistent across applications | <ul><li>Manual configuration is required<li>Auditing configurations is labor intensive</ul> | Provide tools so New Relic configurations can be templated, repeatable, and audited easily |
+| I cannot integrate New Relic into my exiting workflow | <ul><li>Using New Relic in a workflow currently requires significant time investment in custom scripts</ul> | Provide easy to consume tooling that can be leveraged in existing workflows to streamline New Relic usage |
```

Comment:
This is the only problem statement that is in the first person - consider changing to "New Relic cannot be integrated into existing workflows" to make this problem statement structured like the others.